src/services/requestService.js

```diff
@@ -51,4 +52,25 @@ export default class RequestService {
   static async findRequestById(requestId, options = {}) {
     return Request.findByPk(requestId, options);
   }
+
+  /**
+   * Fetches a request instance from the database based on it's primary key within a timeframe.
+   * @static
+   * @param {sting} startDate - Request start date
+   * @param {sting} endDate - Request end date
+   * @param {integer} requestId - Primary key of the request to be fetched.
+   * @returns {Promise<string>} - Number of counts.
+   * @memberof FacilityService
+   */
+  static async search(startDate, endDate, requestId) {
```

Comment:
```suggestion
  static async searchByTime(startDate, endDate, requestId) {
```
I'm asking for this change because there are several search services to be written into the request services, so more specificity is needed.